### ISSUE TITLE
[Bug] [1.19.2] Incompatible with ModernUI

### ISSUE THREAD

**Issue by Meatwo310** [emilyploszaj/emi#243]
MOD Loader: Forge 43.2.17
EMI: emi-1.0.9+1.19.2+forge.jar

## Describe the Bug
EMI and ModernUI would be loaded together,
````
Experiencing delay, is everything alright?
Starting Reload
EMI Reloading...
````
message appears in the lower left corner and no EMI is available at all.

## Reproduction Steps
1. Create a custom profile for Minecraft 1.19.2 | Forge - 43.2.17 using the CurseForge Launcher
2. Install EMI (`emi-1.0.9+1.19.2+forge.jar`) and ModernUI (`ModernUI-1.19.2-3.7.1.1-universal.jar`)
3. Launch this profile, create a new world and open the inventory
4. In the lower right corner, `EMI Reloading...` etc. message appears in the lower right corner and you cannot use EMI.

## Expected Result
You can use the usual EMI features, such as the EMI item list, Favorite list, search bar, and the ability to browse recipes with the `R` key or click.

## Screenshots
![Inventory Screen](https://github.com/emilyploszaj/emi/assets/72017364/3e3d41c5-7920-4c84-bc7a-09c2d329b3cb)
![CurseForge Profile](https://github.com/emilyploszaj/emi/assets/72017364/a35570bb-2131-42cc-bd54-70882d2f6b15)

## Log
https://mclo.gs/41iEas8

## Versions
OS: Windows 11
EMI Version: 1.0.9+1.19.2+forge
Minecraft Version: 1.19.2
ModernUI Version: 1.19.2-3.7.1.1-universal

## Additional Context
Please make an Issue Template...

*Partially used DeepL Translate

**Comment by qznfbnj** [comment]
Related to this question:
https://github.com/BloCamLimb/ModernUI/issues/157#issuecomment-1633798345

---
## Temporary workaround
emi-1.0.4+1.19.2+forge + ModernUI-1.19.2-3.7.1.1-universal -> Reloaded EMI in 1013ms

**Comment by emilyploszaj** [comment]
This has been partially fixed in 1.0.10, there should be no issues in vanilla with these two mods. I have noticed more complex issues involving mods like Create, but there is nothing EMI can do to resolve this issue, I've left a comment on the ModernUI issue with some more context for the developer.

**Comment by Meatwo310** [comment]
I used the vanilla resource pack feature instead of ModernUI.
![image](https://github.com/emilyploszaj/emi/assets/72017364/6d8a906c-435e-4fc0-823f-f6afbd33ad59)

Then it worked, except that the bold fonts appear double.
![image](https://github.com/emilyploszaj/emi/assets/72017364/ebf16b45-5d51-46c2-9543-a36443325d36)

Also, the text seems to display even when using Oculus.
Although not directly related to this issue, I think it is quite a find.

![image](https://github.com/emilyploszaj/emi/assets/72017364/bcd178ee-be4a-4890-86fa-8851d24c5b9d)
![image](https://github.com/emilyploszaj/emi/assets/72017364/89d6a31a-4aec-4575-8aba-d334542f3777)
![image](https://github.com/emilyploszaj/emi/assets/72017364/914e42b2-9a58-4d27-8867-ef53c2f77531)
